Makefile

```
@@ -176,16 +228,7 @@ dev-adopted-rm: dev kind envsubst ## Create adopted cluster deployment
 
 .PHONY: dev-adopted-deploy
 dev-adopted-deploy: dev kind envsubst ## Create adopted cluster deployment
-	@if ! $(KIND) get clusters | grep -q "^$(KIND_CLUSTER_NAME)$$"; then \
-		if [ -n "$(KIND_CONFIG_PATH)" ]; then \
-			$(KIND) create cluster -n $(KIND_CLUSTER_NAME) --config "$(KIND_CONFIG_PATH)" --wait 1m; \
-		else \
-			$(KIND) create cluster -n $(KIND_CLUSTER_NAME) --config "$(PWD)/config/kind-local.yaml" --wait 1m; \
-		fi \
-	fi
-	$(KUBECTL) config use kind-$(KIND_CLUSTER_NAME)
-	$(HELM) repo add kubelet-csr-approver https://postfinance.github.io/kubelet-csr-approver
-	$(HELM) upgrade --install kubelet-csr-approver kubelet-csr-approver/kubelet-csr-approver -n kube-system --set replicas=1
+	make kind-deploy
```

Should we just add `kind-deploy` to prerequisites?

```suggestion
dev-adopted-deploy: dev envsubst kind kind-deploy ## Create adopted cluster deployment
```